api/cmd/bootstrap_test.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"context"
+	"testing"
+
+	"github.com/caraml-dev/mlp/api/pkg/authz/enforcer"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStartKetoBootsrap(t *testing.T) {
+	bootstrapCfg := &BootstrapConfig{
```

Comment:
This test is written wrongly. The test created a new enforcer, which is not the same enforcer used in start keto bootstrap function.